### PR TITLE
Update xampp from 8.1.2-0 to 8.1.4-0

### PR DESCRIPTION
**Pull request by doeber**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by neutric** [comment]
The app bundle is now called `xampp-osx-8.1.4-0-installer.app` and it also has a `Contents/MacOS/osx-x86_64"`, so changing to `executable: "xampp-osx-#{version}-installer.app/Contents/MacOS/osx-x86_64",` in the `installer script` stanza might just do the trick.

File: Casks/xampp.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   end
 
   installer script: {
-    executable: "XAMPP.app/Contents/MacOS/osx-x86_64",
+    executable: "xampp-osx-#{version}-installer.app/Contents/MacOS/osx-x86_64",
```
Comment (SMillerDev) [inline Casks/xampp.rb:17]: Is there an ARM version too?

Comment (SMillerDev) [reply Casks/xampp.rb:17]: Buy is there an ARM executable like this one?

Comment (doeber) [reply Casks/xampp.rb:17]: ah, it's inside the disc image in the installbuilder.sh:

```bash
elif [ "${machine_platform}" == "arm" ];then
 executable=osx-x86_64
```

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @doeber !

**Pull request merged**